examples/pcf8591/main.go

```diff
@@ -0,0 +1,28 @@
+// Connects to a pcf8591 ADC via I2C.
+package main
+
+import (
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers/pcf8591"
+)
+
+var (
+	i2c = machine.I2C0
+)
+
+func main() {
+	machine.I2C0.Configure(machine.I2CConfig{})
+	adc := pcf8591.New(machine.I2C0)
```

Comment:
It would be better to use i2c instead of machine.I2C0.

Reply:
Done.